### PR TITLE
feat: update Go to 1.22.1, update releases

### PR DESCRIPTION
**Pull request by smira**
| Package | Update | Change |
|---|---|---|
| git://sourceware.org/git/elfutils.git | minor | `0.190` -> `0.191` |
| [https://github.com/qemu/qemu.git](https://togithub.com/qemu/qemu) | patch | `8.2.1` -> `8.2.2` |
| [tailscale/tailscale](https://togithub.com/tailscale/tailscale) | patch | `1.60.0` -> `1.60.1` |

### CODE REVIEW

**Comment by frezbo** [comment]
This too:

```cgr.dev/chainguard/wolfi-base 		digest 	09087ed -> 3d6dece```

File: firmware/realtek-firmware/pkg.yaml [hunk 1]
```diff
@@ -11,7 +11,6 @@ steps:
     install:
       - |
         mkdir -p /rootfs/lib/firmware
-        cp -R -p /lib/firmware/RTL8192E /rootfs/lib/firmware
```
Comment (frezbo) [inline firmware/realtek-firmware/pkg.yaml:14]: this got removed?

**Comment by frezbo** [comment]
> This too:
> 
> `cgr.dev/chainguard/wolfi-base digest 09087ed -> 3d6dece`

nvm, didn;t see that

**Review by frezbo — APPROVED** [review]

File: firmware/realtek-firmware/pkg.yaml (hunk 1 above)
Comment (smira) [reply firmware/realtek-firmware/pkg.yaml:14]: yes, removed from linux-firmware

**Comment by smira** [comment]
I will revert back mdadm, as it doesn't build anymore

File: firmware/realtek-firmware/pkg.yaml (hunk 1 above)
Comment (smira) [reply firmware/realtek-firmware/pkg.yaml:14]: it's some super-old WiFi

**Pull request merged**